libraries/rush-lib/src/api/RushConfigurationProject.ts

```diff
@@ -149,6 +149,13 @@ export class RushConfigurationProject {
       );
     }
 
+    if (!semver.valid(this._packageJson.version)) {
```

Comment:
Ensure that `package.json`'s `version` field is in semver format.

Reply:
@elliot-nelson @D4N14L could you test this in your largest monorepos to make sure there aren't any edge cases we're not considering?

Reply:
Tested the new semver error and the "print rush update" warning, both look good.  Also tried some commands like "rush deploy" using 1000RR's branch and everything worked as normal. 👍